### ISSUE TITLE
The referenced tweet is not part of a thread

### ISSUE THREAD

**Issue by jb33k** [Jimmar/NodeThreader#8]
I think there might be a bug somewhere because I get `The referenced tweet is not part of a thread` error when [https://twitter.com/cdixon/status/1427452454543663129](https://twitter.com/cdixon/status/1427452454543663129) which is obvious a thread.

Any ideas?

**Comment by Jimmar** [comment]
Yes, currently I only have the find by search mode and Twitter's search only shows results up to 7 days ago.
I have some code that would be able to recursively get the thread from the last tweet but it's not enabled on the live site yet because it's an expensive operation, I'll fix that at some point.

if you are running your own version, you can activate it by enabling line 63 at https://github.com/Jimmar/NodeThreader/blob/master/server/src/services/threader.js

```js
// cleanedThread = await getThreadTweetsForTweetIdRecursively(tweet_id);
```
